src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -416,54 +190,38 @@ If you also need to define `secrets` in addition to environment variables, you n
 DB_PASSWORD=myPassword
 ```
 
-### Usage
-
-You can access the runtime from Astro components through `Astro.locals` inside any `.astro` file.
+You can access the bindings by using Astro locals like this:
 
 ```astro title="src/pages/index.astro"
 ---
-const runtime = Astro.locals.runtime;
+const { env } = Astro.locals.runtime;
 ---
-
-<pre>{JSON.stringify(runtime.env)}</pre>
 ```
 
 You can access the runtime from API endpoints through `context.locals`:
 
-```js title="src/pages/api/someFile.js"
+```ts title="src/pages/api/someFile.js"
 export function GET(context) {
   const runtime = context.locals.runtime;
 
   return new Response('Some body');
 }
 ```
 
-### Typing
-
-If you have configured `mode: advanced`, you can type the `runtime` object using `AdvancedRuntime`:
+To access the value of the `MY_VARIABLE` binding add the following to your code:
 
-```ts title="src/env.d.ts"
-/// <reference types="astro/client" />
-
-type KVNamespace = import('@cloudflare/workers-types/experimental').KVNamespace;
-type ENV = {
-  SERVER_URL: string;
-  KV_BINDING: KVNamespace;
-};
+```astro title="src/pages/index.astro"
+---
+const { env } = Astro.locals.runtime;
+const myVariable = env.MY_VARIABLE;
+---
+```
 
-type Runtime = import('@astrojs/cloudflare').AdvancedRuntime<ENV>;
+All supported bindings are updated on the Cloudflare docs. You can find the list of supported bindings [here](https://developers.cloudflare.com/workers/wrangler/api/#supported-bindings).
```

Comment:
```suggestion
<ReadMore>See the [list of all supported bindings](https://developers.cloudflare.com/workers/wrangler/api/#supported-bindings) in the Cloudflare documentation.</ReadMore>
```

Vague Link Text Checker doing its job!  :raised_hands:  (The text of the link itself should be descriptive, and inform people of where they'll be taken)